src/test/resources/com/aws/greengrass/secretmanager/config_refresh.yaml

```diff
@@ -0,0 +1,34 @@
+---
+services:
+  main:
+    lifecycle:
+      install: echo "main"
+    dependencies:
+      - aws.greengrass.SecretManager
+      - ComponentRequestingSecrets
+      - ComponentWithNoAccessPolicy
+
+  aws.greengrass.SecretManager:
+    configuration:
+      cloudSecrets:
+        - arn: "arn:aws:secretsmanager:us-east-1:999936977227:secret:randomSecret-74lYJh"
+          labels:
+            - "new"
```

Comment:
Does *only* specifying "new" here mean that "AWS_CURRENT" would not be held in the component? Or is "AWS_CURRENT" always held regardless of what labels are provided? (because I noticed the test case using this config requests and receives "AWS_CURRENT", and not "new"- however it is mocked so that might not be intended

Public docs don't make this clear, I think we could improve that

Reply:
We always download `AWS_CURRENT` label for a secret ARN whether it is configured or not.